browser/src/index.tsx

```diff
@@ -251,6 +251,10 @@ const start = (args: string[]) => {
             return
         }
 
+        if (key === "|") {
+            pluginManager.findAllReferences()
+        }
```

Comment:
Does it have to be a single character?  I'd probably go for something like `g|` since `g` is usually used for the weird extra commands.  That probably wouldn't help you for this method though.

Do we have to map it to a key now?  Could we add it to the command palette instead and then provide a config for mapping keys to Oni commands?  That'd be the more extensible solution so users could map any other commands in the palette to arbitrary keys.